man/leapp.1

```diff
@@ -1,59 +1,165 @@
     By default only error and critical level messages are printed.
 
---verbose
-    Prints all but debug log messages (info, warning, error, critical) to stderr.
+\fB--verbose\fR
+    Print all but debug log messages (info, warning, error, critical) to stderr.
     By default only error and critical level messages are printed.
 
-.SH "LEAPP COMMANDS"
-upgrade
-    Upgrades the current system to the next available major version.
-.SH "ENVIRONMENT VARIABLES"
-LEAPP_CONFIG
+\fB--enable-repo\fR <\fIrepoid\fR>
+    Enable specified repository. Can be used multiple times.
+
+\fB--no-rhsm\fR
+    Skip actions that use Red Hat Subscription Manager. You'll also have to supply custom repositories through \fB--enable-repo\fR (see above).
+
+\fB--whitelist-experimental\fR <\fIActorName\fR>
+    Enable an experimental actor. Can be used multiple times.
+    To use this variable, \fBLEAPP_UNSUPPORTED\fR has to be set. See \fBDeveloper variables\fR for more information.
+
+\fB--reboot\fR
+    (upgrade only) Automatically perform reboot when requested.
+
+\fB--resume\fR
+    (upgrade only) Continue the last execution after it was stopped (e.g. after reboot).
+
+.SS answer
+\fB--add\fR
+    If set, sections will be created even if missing in original answerfile.
+
+\fB--section\fR <\fIdialog_sections\fR>
+    Register answer for a specific section in the answerfile. Can be used multiple times.
+
+
+.SH ENVIRONMENT
+If the argument for the environment variables below is not specified, it is possible to set them to either 1 (true) or 0 (false). They default to 0.
+
+.B LEAPP_CONFIG
+.RS 4
+Overrides the default location of \fIleapp.conf\fR. If not specified, \fI.leapp/leapp.conf\fR is used when the command is executed inside a leapp repository, otherwise the default \fI/etc/leapp/leapp.conf\fR is used.
+.RE
+
+.B LEAPP_LOGGER_CONFIG
+.RS 4
+Overrides the default location of \fIlogger.conf\fR. If not specified, the default \fI/etc/leapp/logger.conf\fR is used.
+.RE
+
+.B LEAPP_DEBUG
+.RS 4
+Enables debug logging. Equivalent to \fB--debug\fR, which takes precedence.
+.RE
+
+.B LEAPP_VERBOSE
 .RS 4
-This environment variable allows to override the default location of leapp.conf. If not specified, .leapp/leapp.conf is used when the command is executed inside a leapp repository, otherwise the default /etc/leapp/leapp.conf is used.
+Enables debug logging. Equivalent to \fB--verbose\fR, which takes precedence.
 .RE
 
-LEAPP_LOGGER_CONFIG
+.B LEAPP_GRUB_DEVICE
 .RS 4
-This environment variable allows to override the default location of logger.conf. If not specified, the default /etc/leapp/logger.conf is used.
+Overrides the automatically detected GRUB core location.
 .RE
 
-LEAPP_DEBUG
+.B LEAPP_NO_RHSM
 .RS 4
-This environment variable enables debug logging, see --debug for more information
+Skip actions that use Red Hat Subscription Manager. Equivalent to \fB--no-rhsm\fR, which takes precedence.
```

Comment:
```suggestion
Skips actions that use Red Hat Subscription Manager. Equivalent to \fB--no-rhsm\fR, which takes precedence.
```
Go through the man page and make the style consistent. Enable/Enables, Perform/Performs, ...

Reply:
Yeah, I consistently use "skip" for options and "skips" for variables.